app.js

```diff
@@ -1,8 +1,112 @@
 // import functions and grab DOM elements
+import { renderPastPolls } from './render-utils.js';
+
+const form = document.querySelector('form');
+const closeButton = document.querySelector('#close-poll');
+const questionELem = document.querySelector('#poll-question');
+const optionATitleElem = document.querySelector('#option-a-title');
+const optionBTitleElem = document.querySelector('#option-b-title');
+const optionAVotesElem = document.querySelector('#option-a-votes');
+const optionBVotesElem = document.querySelector('#option-b-votes');
+const optionAAddButton = document.querySelector('#option-a-add');
+const optionBAddButton = document.querySelector('#option-b-add');
+const optionASubButton = document.querySelector('#option-a-sub');
+const optionBSubButton = document.querySelector('#option-b-sub');
+const pastPollElem = document.querySelector('.past-polls');
```

Comment:
This one has been kind of hard to figure out on my end.  From what I can tell, you are selecting an element with the class of `past-poll` that does not exist in your html.  I know you create this element in your `renderPastPolls` function, but you can't grab it from the DOM if it does not exist yet.  I would create this div in your HTML and then you can append the container div you create in your render function to your hard coded one.